pdfplumber/display.py

```diff
@@ -69,8 +69,6 @@ def get_page_image(
         # Non-modifiable arguments
         prefer_bgrx=True,
     ).to_pil()
-    # In theory `autoclose` when creating it should make it close...
-    # automatically.  In practice this does not seem to be the case.
     pdfium_doc.close()
 
     return img.convert("RGB")
```

Comment:
Oh, by the way, why are you passing `prefer_bgrx=True` but then convert to `RGB` using PIL ?
By omitting this option, you avoid the need to convert. `RGB` should be the default mode you'll get with `page.render(...).to_pil()`.

Reply:
I just skimmed the threads, but sounds mostly historical to me, e.g. (3) was with `wand`, not pypdfium2.